.github/workflows/build-and-test.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       - name: Codecov
         uses: codecov/codecov-action@v2.0.2
         with:
-          files: ./service/build/reports/jacoco/test/jacocoTestReport.xml
+          files: ./service/build/reports/jacoco/test/html/index.html
```

Comment:
`jacoco` won't create this file until the unit tests run so the codecov job fails. I'll open a separate ticket for that!